### PR TITLE
ReadMeUpdate- BigQuery

### PR DESCRIPTION
**Pull request by apurva-birajdar**
Updating 'inclusion sink' filter for BigQuery.

### CODE REVIEW

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 1]
```diff
@@ -65,7 +65,29 @@ To set permissions for the log sink to route to its destination, do the followin
 * If you have owner access to the destination:
    1. add the sink's writer identity to topic and give it the Pub/Sub Publisher role and subscriber role.
    2. add the sink's writer identity to subscription and give it the Pub/Sub subscriber role.
-   3. 
+
+### Create service account credentials
+* Go to the Service accounts section of the IAM & Admin console.
+* Select ```project``` and click ```Create Service Account```.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:71]: For all of the places that you click , change ```codeblock``` to **bold**. 

That is, change the line above to: 
*  Select **project** and click **Create Service Account**.


When referring to code inline, use a single tickmark (`). The triple tickmark (```) creates weird spacing issues when used inline.

**Review by mllezak — COMMENTED** [review]
There's some odd spacing caused by using triple tickmarks inline (such as here):

Click ```Create Key```. 

I suggest replacing triple tickmarks for inline text with either single `tickmarks` or two asterisks **bold**

Other than that, approved.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
 1. [Prerequisites](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-cloud-console)
 2. BigQuery is automatically enabled in new projects. To activate BigQuery in an existing project, enable the BigQuery API.  Please refer to [Enable the BigQuery API](https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service#creating_a_project_and_enabling_the_api) for more information.
 3. Create a BigQuery dataset that stores the data.
-  Please refer to [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
+   Please refer to [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:21]: For more information, see [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets).

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
 1. [Prerequisites](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-cloud-console)
 2. BigQuery is automatically enabled in new projects. To activate BigQuery in an existing project, enable the BigQuery API.  Please refer to [Enable the BigQuery API](https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service#creating_a_project_and_enabling_the_api) for more information.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:19]: For more information, see [Enable the BigQuery API](https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service#creating_a_project_and_enabling_the_api).

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 4]
```diff
@@ -18,15 +18,15 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
...
-  Please refer to [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
+   Please refer to [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
 4. Create a table
     - Expand the: View actions option and click ```Open```.
     - In the details panel, click on (+) Create table.
     - On the Create table page, do the following:
-Enter Table Name. e.g., “user”.
-You can add more fields to the table by clicking  ```+Add Field```.
+      Enter Table Name. e.g., “user”.
+      You can add more fields to the table by clicking  ```+Add Field```.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: To add more fields to the table, click `+Add Field`.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 5]
```diff
@@ -18,15 +18,15 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
...
     - Click ```Create Table```.
-Please refer to [Create Table](https://cloud.google.com/bigquery/docs/tables) for more information
+      Please refer to [Create Table](https://cloud.google.com/bigquery/docs/tables) for more information
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:29]: For more information, see [Create Table](https://cloud.google.com/bigquery/docs/tables).

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 6]
```diff
@@ -18,15 +18,15 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
...
 5. Query table data. Please refer to [Query Table Data](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-cloud-console#query_table_data) for more information
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:30]: For more information, see [Query Table Data](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-cloud-console#query_table_data).

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 1 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:94]: Hi @mllezak 
done.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 7]
```diff
@@ -153,15 +153,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Procedure
 1. On the collector, go to Setup > Tools and Views > Configure Universal Connector.
 2. Enable the universal connector if it is disabled.
-3. Click Upload File and select the relevant plugin based on the version of the Guardium. After it is uploaded, click ```OK```.
+3. Click Upload File and select the relevant plugin based on the version of the Guardium. After it is uploaded, click `OK`.
     * For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip)
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:157]: * For Guardium 11.x, download...

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 8]
```diff
@@ -153,15 +153,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Procedure
 1. On the collector, go to Setup > Tools and Views > Configure Universal Connector.
 2. Enable the universal connector if it is disabled.
-3. Click Upload File and select the relevant plugin based on the version of the Guardium. After it is uploaded, click ```OK```.
+3. Click Upload File and select the relevant plugin based on the version of the Guardium. After it is uploaded, click `OK`.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:156]: Click `Upload File` and select the relevant plug-in based on your version of Guardium. After it is upload, click `OK`.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 9]
```diff
@@ -153,15 +153,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
     * For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip)
     * For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-big_query_guardium_filter.zip)
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:158]: For Guardium 12.x, download...

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 10]
```diff
@@ -153,15 +153,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
-4. Click ```Upload File``` and select the key.json file. After it is uploaded, click ```OK```.
+4. Click `Upload File` and select the key.json file. After it is uploaded, click `OK`.
 5. Click the Plus sign to open the Connector Configuration dialog box.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:160]: Click the plus sign (+) to open...

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 11]
```diff
@@ -153,15 +153,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
 6. Type a name in the Connector name field.
 7. Update the input section to add the details from the [pubsub_big_query.conf](pubsub_big_query.conf) file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
 8. Update the filter section to add the details from the [pubsub_big_query.conf](pubsub_big_query.conf) file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end.
 9. The 'type' fields should match in the input and filter configuration sections. This field should be unique for every individual connector added.
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:164]: Make sure that the `type1 fields match in the input and filter configuration. The field must be unique for every individual connected that you add.

**Review by mllezak — COMMENTED** [review]
Somehow, a bunch of comments got lost -- but here they are again. 

Approved after changes.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 12]
```diff
@@ -18,14 +18,14 @@ This version is compliant with Guardium Data Protection v11.4 and above. Please
 1. [Prerequisites](https://cloud.google.com/bigquery/docs/quickstarts/quickstart-cloud-console)
 2. BigQuery is automatically enabled in new projects. To activate BigQuery in an existing project, enable the BigQuery API.  Please refer to [Enable the BigQuery API](https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service#creating_a_project_and_enabling_the_api) for more information.
 3. Create a BigQuery dataset that stores the data.
-   Please refer to [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
+   For more information, see [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets) for more information
```
Comment (mllezak) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:21]: For more information, see [Create a Dataset](https://cloud.google.com/bigquery/docs/datasets).

**Review by mllezak — COMMENTED** [review]
Sorry -- one more.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 4 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: done

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md (hunk 10 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:160]: done

**Review by mllezak — APPROVED** [review]

**Review by RefaelAdi — APPROVED** [review]

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 13]
```diff
@@ -103,19 +147,21 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 * Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default
 * Download the relevant plugin based on the version of the Guardium.
-1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip)
-2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-big_query_guardium_filter.zip)
+1. For Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip)
+2. For Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-big_query_guardium_filter.zip)
```
[the remark this answers is not in the document]
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md]: done

**Pull request merged**